src/Symfony/Bridge/Twig/Extension/WebLinkExtension.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Extension;
+
+use Fig\Link\Link;
+use Symfony\Component\WebLink\WebLinkManagerInterface;
+
+/**
+ * Twig extension for the Symfony WebLink component.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class WebLinkExtension extends \Twig_Extension
+{
+    private $manager;
+
+    public function __construct(WebLinkManagerInterface $manager)
```

Comment:
Instead of injecting this dep here, it would be better to create a Twig Runtime.